CMakeLists.txt

```diff
@@ -41,16 +45,26 @@ ADD_LIBRARY(common_lib
 		src/common/timer_manager.cc
 		src/common/file.cc
 		src/common/logging.cc)
-ADD_DEPENDENCIES(common_lib asio_ep rest_rpc_ep spdlog_ep )
+ADD_DEPENDENCIES(common_lib asio_ep rest_rpc_ep grpc protobuf spdlog_ep)
 
 # node library
 ADD_LIBRARY(
 		node_lib
 		src/node/node.cc
 		src/log_manager/leader_log_manager.cc
 		src/log_manager/non_leader_log_manager.cc)
-ADD_DEPENDENCIES(node_lib asio_ep rest_rpc_ep common_lib)
-TARGET_LINK_LIBRARIES(node_lib common_lib)
+ADD_DEPENDENCIES(node_lib asio_ep rest_rpc_ep grpc protobuf common_lib)
+
+
+# grpc library
+ADD_LIBRARY(
+		rpc_lib
+		src/rpc/proto.cc
+		${PROTO_SRCS}
+		${GRPC_SRCS}
```

Comment:
Why do we need GRPC_SRCS here? I believe `rpc_lib` only depends on grpc_lib or grpc_lib.so instead of GRPC_SRCS.

Reply:
Removed